src/main/cpp/utils/logging/DragonDataLoggerSignals.h

```diff
@@ -62,7 +62,11 @@ class DragonDataLoggerSignals
         ELECTRICAL_VOLTAGE,
         ELECTRICAL_CURRENT,
         ELECTRICAL_ENERGY,
-        ELECTRICAL_POWER
+        ELECTRICAL_POWER,
+        CACHE_DATA_TV,
+        CACHE_DATA_TX,
```

Comment:
Might want to label these as LIMELIGHT_TV, LIMELIGHT_TX, LIMELIGHT_TY, LIMELIGHT_FIDUCIAL_ID